main/src/com/google/refine/expr/functions/strings/Phonetic.java

```diff
@@ -48,12 +49,33 @@ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
 
 public class Phonetic implements Function {
 
-    // TODO: We could probably lazily initialize these when needed for efficiency
-    static private Metaphone3Keyer metaphone3 = new Metaphone3Keyer();
-    static private DoubleMetaphoneKeyer metaphone2 = new DoubleMetaphoneKeyer();
-    static private MetaphoneKeyer metaphone = new MetaphoneKeyer();
-    static private SoundexKeyer soundex = new SoundexKeyer();
-    static private ColognePhoneticKeyer cologne = new ColognePhoneticKeyer();
+    // TODO deprecate and drop those legacy encodings?
```

Comment:
Soundex is pretty widely used historically (e.g. US Census indexes). I'm not sure how popular it is among OpenRefine users.